.github/workflows/test.yaml

```diff
@@ -27,3 +27,20 @@ jobs:
 
       - name: Test building the docker image
         run: uds run dev-build
+
+  uds_core_integration:
+    runs-on: ubuntu-latest
+    name: Test integration of uds-identity-config and uds-core
+    permissions:
+      pull-requests: read
+      contents: read
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Environment setup
+        uses: defenseunicorns/uds-common/.github/actions/setup@fc12e3a773580020a1d63e254525eab0f8b99fc8
```

Comment:
If you're going to use a SHA here definitely want to leave a comment for which version it came from. 

See: https://github.com/defenseunicorns/uds-package-gitlab/blob/7d31fb6b30c36d65f4c86b635372957e4f28ddc8/.github/workflows/test.yaml#L49